packages/indexer-agent/src/commands/start.ts

```diff
@@ -588,13 +588,30 @@ export default {
       parseGRT(argv.defaultAllocationAmount),
       indexerAddress,
     )
+    const networkSubgraphDeployment = argv.networkSubgraphDeployment
+      ? new SubgraphDeploymentID(argv.networkSubgraphDeployment)
+      : undefined
+    if (networkSubgraphDeployment !== undefined) {
+      // Make sure the network subgraph is being indexed
+      await indexer.ensure(
+        `${networkSubgraphDeployment.ipfsHash.slice(
+          0,
+          23,
+        )}/${networkSubgraphDeployment.ipfsHash.slice(23)}`,
```

Comment:
Because subgraph names are currently limited to `foo/bar` and I think both prefix and suffix have to be shorter than the full IPFS hash would be. We do the same for the subgraphs managed by the indexer agent. It's a bit hacky but we haven't had issues with it so far.